hbase_storage/src/main/java/com/splicemachine/si/data/hbase/coprocessor/TxnLifecycleEndpoint.java

```diff
@@ -16,7 +16,7 @@
 
 import java.io.IOException;
 
-import com.google.common.primitives.Longs;
+import org.spark_project.guava.primitives.Longs;
```

Comment:
i am not saying no to this: but... why?

Reply:
Unification. This change ensures only one, currently prevailing, version of the guava library is used in the module.
See DB-7369 for details. There is some chance once DB-7369 is fixed all imports will be converted back to com.google.common, uniformly this time.